src/borg/item.pyx

```diff
@@ -736,6 +739,16 @@ class ItemDiff:
             mode2 = stat.filemode(self._item2.mode)
             return ({"type": "mode", "old_mode": mode1, "new_mode": mode2}, '[{} -> {}]'.format(mode1, mode2))
 
+    def _time_diffs(self):
+        changes = []
+        attrs = ["ctime", "mtime"]
+        for attr in attrs:
+            if attr in self._item1 and attr in self._item2 and self._item1.get(attr) != self._item2.get(attr):
```

Comment:
you could use the new walrus operator here.

Reply:
I don't think the Cython version we're using supports the use of walrus operators.

Reply:
https://github.com/cython/cython/pull/3691 seems to be in master / cython 3.0 alpha only.